doc-content/enterprise-only/installation/installer-run-proc.adoc

```diff
@@ -141,15 +142,18 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
 . Click *Next* to configure the runtime environment.
 endif::[]
 . When *Processing finished* appears at the top of the screen, click *Next* to complete the installation.
-. If desired, click *Generate Installation Script and Properties File* to save the installation data in an XML file, and then click *Done*. You can use this file to automatically install {PRODUCT} on the same type of server. Note that you must change the `installpath` parameter in the XML file to specify the path of the new server that you want to install {PRODUCT} on. Enter the following command to perform an installation with the XML file:
+. If desired, click *Generate Installation Script and Properties File* to save the installation data in XML files, and then click *Done*.
```

Comment:
I've never been a fan of "if desired", because it sounds strange to my ear, seems like it creates a dangling modifier, and feels ambiguous. For times like this, when it's just an optional step that's up to the user, I typically use "if needed", "optionally", or "If you want to X, ...". I realize a similar argument could be made against "if needed", but to me that seems like less of an accessbility/translatability concern and more widely interpreted. Just my thoughts. Up to you. Whatever you decided, just search "if desired" in the doc. Comes up at least a couple times.

Reply:
OK. Edited out the 19th century poet in me. Changed to Optionally, or in some cases, if required or if needed.